src/DynamoCore/Models/NodeModel.cs

```diff
@@ -1204,8 +1209,10 @@ private void p_PortConnected(PortModel port, ConnectorModel connector)
                 ConnectInput(data, outData, startPort.Owner);
                 startPort.Owner.ConnectOutput(outData, data, this);
                 OnConnectorAdded(connector);
-                ForceReExecuteOfNode = true;
-                OnAstUpdated();
+
+                // Mark node for update
+                //MarkAsDirty();
+                //OnAstUpdated();
```

Comment:
These unwanted changes should be removed (but that means connecting a port will not trigger an evaluation, can you make sure that is desirable)?